domain/build.gradle.kts

```
@@ -0,0 +1,13 @@
+plugins {
+    id("java-library")
+    id("org.jetbrains.kotlin.jvm")
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_7
+    targetCompatibility = JavaVersion.VERSION_1_7
```

```suggestion
    sourceCompatibility = JavaVersion.VERSION_1_8
    targetCompatibility = JavaVersion.VERSION_1_8
```
자바 버전은 1.8 버전을 사용하는 것을 권장합니다. 😄

---

변경했습니다~ 감사합니다!